src/main/starlark/core/repositories/BUILD.com_github_jetbrains_kotlin.bazel

```diff
@@ -11,10 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# dev_io_bazel_rules_kotlin
-load("@{{.KotlinRulesRepository}}//kotlin:jvm.bzl", "kt_jvm_import")
```

Comment:
This needs to provide the old targets with a deprecated warning for at least a version.

The docs have encouraged folks to this repository for a while, so the transition process should be deprecate then remove.